frontend/src/SettingsManager/ExportLogsButton/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { Button, Tooltip } from '@blueprintjs/core';
+
+import { OpenLogsFolder } from '../../../wailsjs/go/app/App';
+
+import './index.css';
+
+export function ExportLogsButton() {
+  return (
+    <Tooltip content="Open logs directory">
+      <Button onClick={OpenLogsFolder} intent="primary" className="export-logs__button">
```

Comment:
The common UX convention is that [primary buttons should actually signal a primary action on a page](https://balsamiq.com/learn/articles/button-design-best-practices/#4-don-t-have-more-than-one-primary-action-button-on-the-screen-at-a-time). I think in this case it's better to remove the prop to align the button with its significance on the page.

Reply:
Got it. Thank you